packages/router/src/__tests__/set.test.tsx

```diff
@@ -35,7 +35,7 @@ test('wraps components in other components', async () => {
   const TestSet = () => (
     <Router useAuth={window.__REDWOOD__USE_AUTH}>
       <Set wrap={[CustomWrapper, GlobalLayout]}>
-        <Route path="/" page={ChildA} name="childa" />
+        <ChildA />
```

Comment:
I'm not sure I understand this change?

Reply:
Now that we don't render multiple routes anymore, I changed it to just render the `<ChildA>` component directly. This is a test of the <Set> component, not the router :)